Makefile

```diff
@@ -138,8 +138,8 @@ check: clean develop check_cyclical modules_boundaries
 	selftests/checkall
 	selftests/check_tmp_dirs
 
-check-long: clean develop check_cyclical modules_boundaries
-	AVOCADO_CHECK_LONG=1 selftests/checkall
+check-full: clean develop check_cyclical modules_boundaries
+	AVOCADO_CHECK_FULL=1 selftests/checkall
```

Comment:
BTW, we're talking about avocado developers that may be "impacted" by the variable name change.  This is not an audience we should try to protect from this type of change.